advanced_functionality/causal-inference/data/heart_failure_clinical_records_dataset.csv

```diff
@@ -0,0 +1,300 @@
+age,anaemia,creatinine_phosphokinase,diabetes,ejection_fraction,high_blood_pressure,platelets,serum_creatinine,serum_sodium,sex,smoking,time,DEATH_EVENT
```

Comment:
@jkroll-aws Is it alright to download entire datasets into the repo or should the notebook download it from S3?